tests/test_broker.py

```diff
@@ -25,6 +25,7 @@ async def multiple_subscribers(self):
 
         async def tester():
             address = await broker.getsockname()
+            address = address[:2]  # IPv6 gives 4-tuple, need only host, port
```

Comment:
That is easy to do (and a good idea) 
The suggestion above gives a "TypeError: 'coroutine' object is not subscriptable" so it will be something like:
```python
async def get_broker_address(broker):
    address_tuple = await broker.getsockname()
    # IPv6 gives 4-tuple, need only host and port.
    return address_tuple[:2]
```
?

Reply:
Ok!